<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Idesk-x-set</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Создание иконок</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Command">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Исполняемая комманда</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Idesk_CaptionTip">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Icon">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Idesk_Caption">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>361</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Координаты на рабочем столе</string>
      </property>
      <widget class="QSpinBox" name="Idesk_X">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximum">
        <number>800</number>
       </property>
       <property name="value">
        <number>600</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="Idesk_Y">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>361</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Размер иконки на рабочем столе</string>
      </property>
      <widget class="QSpinBox" name="Idesk_Width">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>48</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="Idesk_Height">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>30</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>48</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ширина</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Высота</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="Idesk_Command">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Название иконки</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Картинка для иконки</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Исходный desktop-файл</string>
      </property>
      <widget class="QLineEdit" name="Desktop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Desktop">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>30</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Обзор</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Название типа</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Idesk_Icon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Управление настройками</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
